011_strnlen-compile-warning-fix.patch
PHP_5_2
http://cvs.php.net/viewvc.cgi/php-src/main/spprintf.c?r1=1.25.2.2.2.15&r2=1.25.2.2.2.16&diff_format=u
http://bugs.php.net/bug.php?id=47831
Fixed bug #47831 (Compile warning for strnlen() in main/spprintf.c)

diff -r d409743914d7 -r a5eccb8408e8 main/spprintf.c
--- a/main/spprintf.c	Fri Apr 10 11:39:27 2009 +0200
+++ b/main/spprintf.c	Fri Apr 10 11:41:21 2009 +0200
@@ -76,6 +76,7 @@
  * SIO stdio-replacement strx_* functions by Panos Tsirigotis
  * <panos@alumni.cs.colorado.edu> for xinetd.
  */
+#define _GNU_SOURCE
 #include "php.h"
 
 #include <stddef.h>
